tools/setup/emoji/emoji_names.py

```diff
@@ -1,1580 +1,2055 @@
+    "1f1f9-1f1ff": {"canonical_name": "flag: Tanzania", "aliases": []},
+    "1f1fa-1f1e6": {"canonical_name": "flag: Ukraine", "aliases": []},
+    "1f1fa-1f1ec": {"canonical_name": "flag: Uganda", "aliases": []},
+    "1f1fa-1f1f2": {"canonical_name": "flag: U.S. Outlying Islands", "aliases": []},
+    "1f1fa-1f1f3": {"canonical_name": "flag: United Nations", "aliases": []},
+    "1f1fa-1f1f8": {"canonical_name": "flag: United States", "aliases": []},
+    "1f1fa-1f1fe": {"canonical_name": "flag: Uruguay", "aliases": []},
+    "1f1fa-1f1ff": {"canonical_name": "flag: Uzbekistan", "aliases": []},
+    "1f1fb-1f1e6": {"canonical_name": "flag: Vatican City", "aliases": []},
+    "1f1fb-1f1e8": {"canonical_name": "flag: St. Vincent & Grenadines", "aliases": []},
+    "1f1fb-1f1ea": {"canonical_name": "flag: Venezuela", "aliases": []},
+    "1f1fb-1f1ec": {"canonical_name": "flag: British Virgin Islands", "aliases": []},
+    "1f1fb-1f1ee": {"canonical_name": "flag: U.S. Virgin Islands", "aliases": []},
+    "1f1fb-1f1f3": {"canonical_name": "flag: Vietnam", "aliases": []},
+    "1f1fb-1f1fa": {"canonical_name": "flag: Vanuatu", "aliases": []},
+    "1f1fc-1f1eb": {"canonical_name": "flag: Wallis & Futuna", "aliases": []},
+    "1f1fc-1f1f8": {"canonical_name": "flag: Samoa", "aliases": []},
+    "1f1fd-1f1f0": {"canonical_name": "flag: Kosovo", "aliases": []},
+    "1f1fe-1f1ea": {"canonical_name": "flag: Yemen", "aliases": []},
+    "1f1fe-1f1f9": {"canonical_name": "flag: Mayotte", "aliases": []},
+    "1f1ff-1f1e6": {"canonical_name": "flag: South Africa", "aliases": []},
+    "1f1ff-1f1f2": {"canonical_name": "flag: Zambia", "aliases": []},
+    "1f1ff-1f1fc": {"canonical_name": "flag: Zimbabwe", "aliases": []},
+    "1f201": {"canonical_name": 'Japanese "here" button', "aliases": ['"here"\u30b3\u30b3']},
+    "1f202": {
+        "canonical_name": 'Japanese "service charge" button',
+        "aliases": ['"service charge"', "\u30b5"],
     },
-    # hurt and injured from https://beebom.com/emoji-meanings/. Chose hurt as
-    # primary since I think it can cover a wider set of things (e.g. emotional
-    # hurt)
-    "1f915": {"canonical_name": "hurt", "aliases": ["head_bandage", "injured"]},
-    # devil from https://emojipedia.org/smiling-face-with-horns/,
-    # smiling_face_with_horns from gemoji/Unicode
-    "1f608": {
-        "canonical_name": "smiling_devil",
-        "aliases": ["smiling_imp", "smiling_face_with_horns"],
+    "1f21a": {"canonical_name": "japanese_free_of_charge_button", "aliases": ["u7121"]},
+    "1f22f": {"canonical_name": 'Japanese "reserved" button', "aliases": ['"reserved"', "\u6307"]},
+    "1f232": {"canonical_name": "japanese_prohibited_button", "aliases": ["u7981"]},
+    "1f233": {"canonical_name": 'Japanese "vacancy" button', "aliases": ['"vacancy"', "\u7a7a"]},
+    "1f234": {"canonical_name": "japanese_passing_grade_button", "aliases": ["u5408"]},
```

Comment:
I think rather than override the directional quotes to straight quotes, it might be better to just delete the quotes (and have the script do that.)

Reply:
I love it when scripts do things. Thanks for the suggestion :)